docs/setting-up/client/mongodb.md

```diff
@@ -180,13 +180,18 @@ pmm-admin add mongodb --tls \
 --tls-certificate-key-file=PATHTOCER \
 --tls-certificate-key-file-password=IFPASSWORDTOCERTISSET \
 --tls-ca-file=PATHTOCACERT
+--authentication-mechanism=AUTHENTICATION-MECHANISM
+--authentication-database=AUTHENTICATION-DATABASE
 ```
 
 where:
 
 - `PATHTOCERT`: Path to TLS certificate file.
 - `IFPASSWORDTOCERTISSET`: Password for TLS certificate file.
 - `PATHTOCACERT`: Path to certificate authority file.
+- `AUTHENTICATION-MECHANISM`: Authentication mechanism. Default is empty. Use MONGODB-X509 for ssl certificates.
+- `AUTHENTICATION-DATABASE`: Authentication database. Default is empty. Use $external for ssl certificates.
```

Comment:
Is `$external` a literal string or a variable expansion? Are there any issues with quoting? Can you give an example of use. Plus, as above, what other values are possible and where are they found?

Reply:
```suggestion
- `AUTHENTICATION-DATABASE`: Authentication database. Default is empty. Use `$external` for SSL certificates.
```